src/content/docs/references/v2/acl.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: Access Control List
```

Comment:
Imho we're using a capability list (or access control matrix, depending on the viewpoint) and should be consistent in our naming.

Some vaguely relevant [reference](https://www.geeksforgeeks.org/difference-between-access-control-list-and-capability-list/) and another [reference](https://identitymanagementinstitute.org/access-control-matrix-and-capability-list/).
I think this should also be reflected in the tauri code as well.

Reply:
let's do it (i've missed you 😂)

Reply:
Shall I address this in the draft PR for the concepts docs? #861

Reply:
Actually after reading more into the wiki definitions of [ACL](https://en.wikipedia.org/wiki/Access-control_list) and [Capability](https://en.wikipedia.org/wiki/Capability-based_security) I tend to agree that ACL is still the simplest description for the whole architecture.

> an access-control list (ACL) is a list of permissions[[a]](https://en.wikipedia.org/wiki/Access-control_list#cite_note-1) associated with a [system resource](https://en.wikipedia.org/wiki/System_resource) (object or facility)

We have some overlapping between both concepts but the above definition is still the most fitting, as we handle objects (windows) and resources (commands/interfaces/permissions).